src/App.js

```diff
@@ -1,5 +1,118 @@
+import { Console, Random } from "@woowacourse/mission-utils";
+
 class App {
-  async play() {}
+  async play() {
+    Console.print("숫자 야구 게임을 시작합니다.");
+    let reStart;
+    do {
+      reStart = await this.playGame();
+    } while (reStart === "1");
+  }
+
+  verifyInputFormat(input) {
+    const USER_INPUT = input;
+    if (USER_INPUT === "") {
+      throw new Error(
+        "[ERROR]: 아무것도 입력하지 않았습니다. 서로 다른 세 자리 숫자를 입력해 주세요."
+      );
+    }
+    if (isNaN(USER_INPUT)) {
+      throw new Error(
+        "[ERROR]: 문자를 입력하였습니다. 서로 다른 세 자리 숫자를 입력해 주세요."
+      );
+    }
+    if (!USER_INPUT.trim()) {
+      throw new Error(
+        "[ERROR]: 공백없이 입력해야 합니다. 서로 다른 세 자리를 입력해 주세요."
+      );
+    }
+    if (USER_INPUT.length != 3) {
+      throw new Error(
+        "[ERROR]: 길이가 세 자리가 아닙니다. 서로 다른 세 자리 숫자를 입력해 주세요."
+      );
+    }
+    if (!this.areDigitsUnique(USER_INPUT)) {
+      throw new Error(
+        "[ERROR]: 자릿수가 달라야 합니다. 서로 다른 세 자리를 입력해 주세요."
+      );
+    }
+  }
+
+  areDigitsUnique(input) {
+    const DIGITS = input.split("");
+    return new Set(DIGITS).size === DIGITS.length;
```

Comment:
저도 미션하면서 알게 된 사실인데 문자열도 iterable object여서 아래와 같이 바로 작성하는 것이 가능합니다!

```suggestion
    return new Set(input).size === input.length;
```

즉, split해서 배열로 변환하는 과정 없이도 바로 넣을 수 있습니다!!!

Reply:
짚어주셔서 감사합니다!!! 아직 자바스크립트 공부가 제가 많이 부족하네요!!🤣